app/code/Magento/User/Controller/Adminhtml/Auth/Forgotpassword.php

```diff
@@ -7,26 +7,38 @@
 namespace Magento\User\Controller\Adminhtml\Auth;
 
 use Magento\Security\Model\SecurityManager;
+use Magento\Framework\App\ObjectManager;
```

Comment:
Please import all used classes in this file.

Reply:
Done.